### ISSUE TITLE
Select-Wrapper: Accessibility Issue

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#1195]
Description  
---
If Select-Wrapper component is tested with AXE Browser extension, the following error occurs:
"_Ensures every ARIA input field has an accessible name_"

That's because of a false usage of the `aria-labelledby` attribute of the fake-option-list drop down. Currently the value of this attribute is the value of the `label`, but should be an ID of the label (which is currently not available).

Possible solution: Add an ID to the `label` and replace current `aria-labelledby` value with this ID.

To Reproduce
---
1. Go to https://designsystem.porsche.com/latest/#/patterns/forms/example-create-account 
2. Open Dev Tools
3. Install AXE Browser extension
4. Scan page

Expected behavior
---
No accessibility issues

Test Environment
---
Platform/Browser/Version: All, AXE Browser extension
